src/components/Playlist/index.js

```diff
@@ -24,7 +25,67 @@ const CategoryIcon = ({icon}) => {
   return <img src={icon} className='ml2 mt1 h1' alt='' />
 }
 
-const PlaylistItem = ({item, extraClasses}) => {
+const StyledPlaylistItem = styled.li`
+	&:before, &:after {
+    content: '';
+    position: absolute;
+    transition: 150ms;
+	}
+	&:before {
+    width: 9px;
+    height: 9px;
+    border-radius: 50%;
+    border: 1px solid var(--light-gray);
+    background: var(--tag-gray);
+    box-shadow: 0 0 0 1px var(--tag-gray);
+    left: 20px;
+    top: 22px;
+    z-index: 2;
+	}
+	&:after {
+    width: 1px;
+    height: 100%;
+    border-width: 0 0 0 1px;
+    border-style: solid;
+    border-color: var(--light-gray);
+    left: 24px;
+    top: 32px;
+    z-index: 1;
+	}
+	&:hover:before {
+    background: var(--white);
+    box-shadow: 0 0 0 1px var(--white);
+	}
+	&.viewed {
+    color: var(--gray);
+	}
+  &.viewed:before {
+    border: 1px solid ${props => props.card === 'playlist' ? 'var(--turquoise)' : 'var(--orange)'};
+    background: ${props => props.card === 'playlist' ? 'var(--turquoise)' : 'var(--orange)'};
+  }
+  &.viewed:after {
+    border-color: ${props => props.card === 'playlist' ? 'var(--turquoise)' : 'var(--orange)'};
+  }
+	&.pre-next:after {
+		border-style dashed;
+	}
+  &.next:before {
+    left: 21px;
+    border-radius: 0;
+    width: 0;
+    height: 0;
+    border-style: solid;
+    border-width: 6px 0px 6px 9px;
+    border-color: transparent transparent transparent ${props => props.card ? 'var(--turquoise)' : 'var(--orange)'};
```

Comment:
@joelhooks In this case I'm just using `card` since this is the actual `Playlist` file... This line is basically saying "If `card` is present, it's the `<PlaylistCard>` so use turquoise, otherwise use orange". However, it would probably be good to add `=== 'playlist'` to be more clear, since the expanded `CourseCard` has a playlist (with orange).